prime-router/docs/playbooks/how-to-onboard-county-receiver.md

```diff
@@ -0,0 +1,79 @@
+# How to Onboard a County Receiver
+
+## Introduction
+Most of the receivers we have added to ReportStream are state public health departments, who typically have more
+resources at their disposal, and systems that we send data to directly. Counties differ in a few ways:
+
+1. They will usually want to receive CSV
+2. They will want to use the download site instead of receiving the data directly
+
+To that end, I have created a quick and dirty guide to adding a county receiver.
+
+## Steps
+1. Collect the names of the people who will be logging into the download site for the county. It may only be one person, or a small handful.
```

Comment:
"Collect the names and email addresses of the people..."

The people may not be on the email chain so the person wouldn't have access to their email